src/Raven.Client/Documents/Operations/Attachments/AttachmentDetails.cs

```diff
@@ -13,4 +19,16 @@ public class AttachmentName
         public string ContentType;
         public long Size;
     }
-}
\ No newline at end of file
+
+    public class AttachmentRequest
+    {
+        public AttachmentRequest(string documentId, string name)
+        {
+            DocumentId = documentId;
```

Comment:
check both of them for null and whitespace?
Both of them are mandatory?

Reply:
done